internal/core/command/controller/messaging/utils.go

```diff
@@ -25,29 +26,29 @@ import (
 
 // validateRequestTopic validates the request topic by checking the existence of device and device service,
 // returns the internal device request topic to which the command request will be sent.
-func validateRequestTopic(prefix string, deviceName string, commandName string, method string, dic *di.Container) (string, error) {
+func validateRequestTopic(prefix string, deviceName string, commandName string, method string, dic *di.Container) (string, string, error) {
```

Comment:
please update the docs comments above to include the new 2nd return from this function

Reply:
updated